content/en/blog/2024/docs-localized.md

```diff
@@ -0,0 +1,42 @@
+---
+title: OpenTelemetry website goes multilingual!
+linkTitle: Multilingual website
+date: 2024-08-21
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco),
+issue: 4863
+sig: Comms
+cSpell:ignore: Comms
+---
+
+We are happy to announce that the OpenTelemetry documentation now supports
+multiple languages! Teams of contributors have already started to translate our
+resources into Portuguese (pt), Chinese (zh), and Japanese (ja).
+
+The OpenTelemetry project has grown to include contributors and users from all
+over the world. Making our documentation available in multiple languages is an
+important step for ensuring that everyone, regardless of their native language,
+can contribute to the project. We are also thrilled that this initiative gives
+end users access to multilingual documentation, making it easier to learn and
+understand OpenTelemetry.
+
+To access the documentation in your preferred language, use the language
+selector at the top right of the page.
+
+This is just the beginning! A team of contributors has just begun to translate
+the documentation into Spanish (es), and we expect to add more languages in the
+future.
+
+We invite you to contribute to this effort! You can find fellow localization
+contributors in the
+[#otel-docs-localization](https://cloud-native.slack.com/archives/C076RUAGP37)
+Slack channel. If you are fluent in a language that is supported already, you
+can help by authoring or reviewing translations. If your language is not yet
+supported and you'd like to help translate, please reach out to us by raising an
+issue.
```

Comment:
Add a link to the new-issue GH feature and pre-populate it with at least a title? WDYT?

Reply:
yes! Thanks